bioio_ome_zarr/tests/writers/test_ome_zarr_writer.py

```diff
@@ -313,25 +313,25 @@ def run() -> None:
 
 
 @pytest.mark.parametrize(
-    "shape, chunk_size, shard_factor",
+    "shape, chunk_size, shard_shape",
     [
         # 2D YX, shard 2x2
-        ((16, 16), (4, 4), (2, 2)),
+        ((16, 16), (4, 4), [(4, 4)]),
         # 3D TYX, shard 1x2x2
-        ((2, 16, 16), (1, 4, 4), (1, 2, 2)),
+        ((2, 16, 16), (1, 4, 4), [(1, 8, 8)]),
         # 4D CZYX, shard 1x1x2x2
-        ((2, 2, 16, 16), (1, 1, 4, 4), (1, 1, 2, 2)),
+        ((2, 2, 16, 16), (1, 1, 4, 4), [(2, 2, 16, 16)]),
         # 4D TZYX, shard 1x1x2x2
-        ((3, 2, 16, 16), (1, 1, 4, 4), (1, 1, 2, 2)),
+        ((3, 2, 16, 16), (1, 1, 4, 4), [(1, 2, 16, 16)]),
         # 5D TCZYX, shard 1x1x1x2x2
-        ((2, 2, 2, 16, 16), (1, 1, 1, 4, 4), (1, 1, 1, 2, 2)),
+        ((2, 2, 2, 16, 16), (1, 1, 1, 4, 4), [(2, 2, 2, 16, 16)]),
```

Comment:
the comments above these look inconsistent

Reply:
resolved in [f51dd33](https://github.com/bioio-devs/bioio-ome-zarr/pull/86/commits/f51dd332dbbe29ce03425ba59652417db98e0737)